src/ILCompiler.ReadyToRun/src/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -907,6 +910,11 @@ private static bool IsDynamicStatics(TypeDesc type)
                     // OR 2. no code sharing lookup in call
                     // OR 3. we have have resolved to an instantiating stub
 
+                    // This check for introducing an instantiation stub comes from the logic at the very end of 
+                    // MethodTable::TryResolveConstraintMethodApprox under the conditional block
+                    // if (!pMD->GetMethodTable()->IsInterface()) around line 10050 in the file vm/methodtable.cpp.
+                    useInstantiatingStub = !directMethod.OwningType.IsInterface && directMethod.OwningType.IsValueType;
```

Comment:
If `IsValueType` is true, `IsInterface` will never be true. We can drop the `IsInterface` check.

Reply:
Thanks for pointing that out, it actually occurred to me but I'm still fighting with finding the sweet spot in CoreCLR ports akin to what we discussed in the past - diverging more substantially from the CoreCLR C++ code can presumably clean up bits of getCallInfo and other JIT interface methods but the maintenance cost of diverging is high - or, even worse, next to impossible to even evaluate because of the infinitely subtle nature of these interface methods. From a 30K feet view perspective I would dare say that CPAOT has been (and to some extent still is) about properly implementing getCallInfo.